config/routes.rb

```diff
@@ -5,8 +5,9 @@
   resources 'items', only: [:index]
 
   resources :carts, only: [:create]
```

Comment:
@tmikeschu I think this might be my doing originally but should we change it to `resources :carts, only: [:create, :index]` and to make the routes follow conventions better? So it would be `carts#index`.